lib/assessment.rb

```diff
@@ -13,28 +13,32 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    reduce (memo, f) { return memo + f }
   end
 
   # Question 3
   # Write ruby code that returns true if some of the entries
   # in the numbers enumerable are odd
+
+  # Yeah... confused
   def some_odd(numbers)
-    # Your code goes here
+    each_with_index { |number, index| number.odd? }
   end
 
   # Question 4
   # Write ruby code that returns true if every entry in the
   # numbers enumerable are even
+
+  # same comment as above
   def every_even(numbers)
-    # Your code goes here
+    each_with_index { |number, index|  number.even? }
   end
 
   # Question 5
   # Write ruby code that returns an array with all of the
   # entries from the words enumerable capitalized.
   def transform(words)
-    # Your code goes here
+    upcase
```

Comment:
If we want to perform a transformation on every element in a collection, which enumerable method would we want to use?

As far as the transformation itself, `upcase` will turn "apple" into "APPLE", but what we want is "Apple". Are there any string methods we could use to do that?